cli/utils/response.go

```diff
@@ -1,4 +1,4 @@
-package cli
+package utils
```

Comment:
Additionally, if the "client" can just be created once and re-used, then injected into commands (rather than globally referenced) writing tests will be much, much easier. (Not saying: make this change now, but it will eventually need to happen)